crates/ruff_linter/src/rules/refurb/rules/reimplemented_operator.rs

```diff
@@ -213,10 +213,20 @@ fn subscript_slice_to_string<'a>(expr: &Expr, locator: &Locator<'a>) -> Cow<'a,
     if let Expr::Slice(expr_slice) = expr {
         Cow::Owned(slice_expr_to_slice_call(expr_slice, locator))
     } else if let Expr::Tuple(tuple) = expr {
+        let inner = tuple
+            .elts
+            .iter()
+            .map(|expr| match expr {
+                Expr::Slice(expr) => Cow::Owned(slice_expr_to_slice_call(expr, locator)),
+                _ => Cow::Borrowed(locator.slice(expr)),
+            })
+            .collect::<Vec<_>>()
+            .join(", ");
```

Comment:
Presumably we'll drop some trivia and fixes won't retain verbatim spacing and such? We could consider only doing this if a `:` is in the elements?

Reply:
I think that would be nice.